test/complex.jl

```diff
@@ -15,6 +15,17 @@ end
 @test sprint(show, complex(1, 0), context=:compact => true) == "1+0im"
 @test sprint(show, complex(true, true)) == "Complex(true,true)"
 
+@testset "unary operator on complex boolean" begin
+    @test +Complex(true, true) === complex(1, 1)
+    @test +Complex(true, false) === complex(1, 0)
+    @test +Complex(false, true) === complex(0, 1)
+    @test +Complex(false, false) === complex(0, 0)
+    @test -Complex(true, true) === complex(-1, -1)
+    @test -Complex(true, false) === complex(-1, 0)
+    @test -Complex(false, true) === complex(0, -1)
+    @test -Complex(false, false) === complex(0, 0)
```

Comment:
Actually, you're going to hate me, but the RHS should be `Complex(0, 0)` not `complex(0, 0)`.

Reply:
Ok! It's cool!

Reply:
Actually, now life has become pretty smooth after learning `git` commands properly, to modify anything on branches. Thanks for putting up with me during my initial days!

Reply:
Btw, `complex(x::Real, y::Real) = Complex(x, y)` is defined in `complex.jl` so it should not have mattered!